src/api/__specs__/campaigns-spec.js

```diff
@@ -5,12 +5,21 @@ import * as getJSONP from '../../lib/getJSONP';
 describe('campaigns', () => {
   const callback = sinon.spy();
 
-  beforeEach(() => {
+  before(() => {
     sinon.stub(getJSONP, 'default');
   });
 
-  afterEach(() => {
+  after(() => {
     getJSONP.default.restore();
+  })
+
+  afterEach(() => {
+    try{
+      getJSONP.default.reset();
```

Comment:
Yeah, does this work? I thought `reset` was only available for `sinon` spies not stubs? 😲